akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/ActorContextDelegateSpec.scala

```diff
@@ -29,20 +29,20 @@ object ActorContextDelegateSpec {
       case Ping =>
         monitor ! ResponseFrom(PingTag, Ping)
         Behaviors.same
-      case msg @ Pong =>
```

Comment:
Hmm, why isn't it just `case Pong` if it is a singleton?

Reply:
I assumed the reasoning was to protect against accidentally delegating a different message to the other behavior below (e.g. when refactoring) - I'd also be fine with `case Pong`

Reply:
Not important, this is fine.

Reply:
this would be ok, but since we already have other cases here that is using Pong - Pong, I think I prefer https://github.com/akka/akka/pull/31509